Tools/autotest/param_metadata/wikiemit.py

```diff
@@ -17,56 +19,53 @@ def __init__(self):
         = Vehicles =    
         '''
         self.f.write(preamble)
-    
-    def close(self): 
-        self.f.close
-    
+
+    def close(self):
+        self.f.close()
+
     def camelcase_escape(self, word):
         if re.match(r"([A-Z][a-z]+[A-Z][a-z]*)", word.strip()):
-            return "!"+word
+            return "!" + word
         else:
             return word
-    
+
     def wikichars_escape(self, text):
         for c in "*,{,},[,],_,=,#,^,~,!,@,$,|,<,>,&,|,\,/".split(','):
-            text = re.sub("\\"+c, '`'+c+'`', text)
+            text = re.sub("\\" + c, '`' + c + '`', text)
         return text
-    
+
     def emit_comment(self, s):
         self.f.write("\n\n=" + s + "=\n\n")
 
     def start_libraries(self):
         self.emit_comment("Libraries")
-        
-    def emit(self, g, f):    
-        
-        t = "\n\n== %s Parameters ==\n"  % (self.camelcase_escape(g.name))
-        
+
+    def emit(self, g, f):
+
+        t = "\n\n== %s Parameters ==\n" % (self.camelcase_escape(g.name))
```

Comment:
I haven't checked all places but I think most don't leave a space after function definition so let's remove it here